src/theory/theory.cpp

```diff
@@ -23,6 +23,7 @@
 
 #include "base/check.h"
 #include "expr/node_algorithm.h"
+#include "options/smt_options.h"
```

Comment:
Why is this needed?

Reply:
Because there is a call to `options::produceModels()` in the file. This was not necessary before because the inclusion-chain to `theory.h` had `smt_options.h` before (not sure where) but does not any longer.

Reply:
Makes sense.